test/acceptance/go.mod

```diff
@@ -9,5 +9,6 @@ require (
 	github.com/gruntwork-io/terratest v0.34.6
 	github.com/hashicorp/consul/api v1.12.0
 	github.com/hashicorp/consul/sdk v0.9.0
+	github.com/hashicorp/lint-consul-retry v1.2.0 // indirect
```

Comment:
I don't think this should be here. If you run `go mod tidy`, does it remove this? If it does, please update the PR to remove this 🙂 

(We should probably have CI check for this 🤔 )

---
What `go mod tidy` does is, basically, examine the source code in the current module to figure out what libraries it imports. It downloads those libraries, if necessary, and updates go.mod and go.sum. So it's always good to run `go mod tidy` if you notice your PR has unexpected changes to go.mod or go.sum (see [Go Modules](https://go.dev/blog/using-go-modules) for more).

When you run `go get github.com/hashicorp/lint-consul-retry`, it updates go.mod and go.sum with that library. Instead, you can run `go install github.com/hashicorp/lint-consul-retry@master`, and it will install the binary to your GOPATH:

```shell
$ go install github.com/hashicorp/lint-consul-retry@master
$ go env GOPATH
/Users/pglass/go
$ ls $(go env GOPATH)/bin/lint-consul-retry
/Users/pglass/go/bin/lint-consul-retry
```